DataCapturing/Persistence/BinarySerializer.swift

```diff
@@ -0,0 +1,299 @@
+//
```

Comment:
This is the new version of the CyfaceBinaryFormatSerializer I have renamed (see comment a few files above).